### PR TITLE
CY-3420 Support DELETE calls returning 204

### PR DESCRIPTION
**Pull request by tehasdf**

### CODE REVIEW

File: cloudify_rest_client/events.py [hunk 1]
```diff
@@ -100,7 +100,8 @@ def delete(self, deployment_id, include_logs=False, message=None,
                                     deployment_id=deployment_id,
                                     **kwargs)
 
-        response = self.api.delete(uri, params=params)
+        response = self.api.delete(uri, params=params,
+                                   expected_status_code=200)
```
Comment (geokala) [inline cloudify_rest_client/events.py:104]: Not 204?

File: cloudify_rest_client/executions.py [hunk 2]
```diff
@@ -338,5 +338,6 @@ def delete(self, to_datetime=None, keep_last=None, **kwargs):
             data['keep_last'] = keep_last
         response = self.api.delete('/{self._uri_prefix}'.format(self=self),
                                    data=data,
-                                   params=kwargs)
+                                   params=kwargs,
+                                   expected_status_code=200)
```
Comment (geokala) [inline cloudify_rest_client/executions.py:342]: Not 204?

File: cloudify_rest_client/events.py (hunk 1 above)
Comment (tehasdf) [reply cloudify_rest_client/events.py:104]: in events and executions, we do return the deleted count. Which is, as you know, allowed by the rfc.

**Review by geokala — DISMISSED** [review]

File: cloudify_rest_client/events.py (hunk 1 above)
Comment (tehasdf) [reply cloudify_rest_client/events.py:104]: CY-3535 is about updating the rest docs to reflect all these changes

**Review by geokala — DISMISSED** [review]

**Review by geokala — APPROVED** [review]
Any more for any more?

**Pull request merged**